src/Type/Php/RandomIntFunctionReturnTypeExtension.php

```diff
@@ -16,11 +17,15 @@ class RandomIntFunctionReturnTypeExtension implements \PHPStan\Type\DynamicFunct
 
 	public function isFunctionSupported(FunctionReflection $functionReflection): bool
 	{
-		return $functionReflection->getName() === 'random_int';
+		return in_array($functionReflection->getName(), ['random_int', 'rand']);
```

Comment:
I figured functionality wise, it would be easiest to use the same ReturnTypeExtension for `rand` and `random_int`.

I did not change the class-name to not break BC